### ISSUE TITLE
require.context in nested modules

### ISSUE THREAD

**Issue by goodmind** [storybooks/storybook#3406]
### Bug or support request summary

`require.context` only polyfilled in top config, but not in other required file in `@storybook/addon-storyshots`

**Comment by quinn** [comment]
I think I am having this same issue:

```
$ node_modules/.bin/jest
 => Loading custom .babelrc
 FAIL  stories/test.js
  ● Test suite failed to run

    TypeError: require.context is not a function

      1 | import { configure } from '@storybook/react';
      2 |
    > 3 | const context = require.context('../components', true, /stories.js$/)
        |                         ^
      4 |
      5 | configure(() => context.keys().forEach(context), module)
      6 |

      at Object.<anonymous> (stories/index.js:3:25)
      at newRequire (node_modules/@storybook/addon-storyshots/dist/require_context.js:83:14)
      at loadStories (evalmachine.<anonymous>:6:2)
      at ConfigApi.configure (node_modules/@storybook/core/dist/client/preview/config_api.js:63:9)
      at evalmachine.<anonymous>:9:22
      at runWithRequireContext (node_modules/@storybook/addon-storyshots/dist/require_context.js:113:3)
      at Object.load (node_modules/@storybook/addon-storyshots/dist/react/loader.js:33:33)
      at loadFramework (node_modules/@storybook/addon-storyshots/dist/frameworkLoader.js:36:17)
      at testStorySnapshots (node_modules/@storybook/addon-storyshots/dist/index.js:72:54)
      at Object.<anonymous> (stories/test.js:3:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.854s
Ran all test suites.
```

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.10`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.10)
